scripts/test_remote_connections_real.py

```
@@ -0,0 +1,349 @@
+        session.add(conn_a)
+        path_b = MonitoredPath(name="Hot B", source_path=str(INSTANCE_B_DIR / "hot"))
+        path_b.storage_locations.append(cold_b)
+        session.add(path_b)
+        session.commit()
+        path_b_id = path_b.id
+        conn_a_id = conn_a.id
+
+    print("\n--- Step 3: Starting Uvicorn Instances (Scheduler Disabled) ---")
+    proc_a, log_a = await start_instance("A", PORT_A, INSTANCE_A_DIR)
+    proc_b, log_b = await start_instance("B", PORT_B, INSTANCE_B_DIR)
+    if not proc_a or not proc_b: return
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)